www/gatsby-node.js

```diff
@@ -9,6 +9,11 @@ const url = require(`url`)
 const getpkgjson = require(`get-package-json-from-github`)
 const parseGHUrl = require(`parse-github-url`)
 const { GraphQLClient } = require(`graphql-request`)
+const yaml = require("js-yaml")
```

Comment:
Did you look at pulling this in using `gatsby-transformer-yaml` instead? This shouldn't be necessary.

Reply:
https://github.com/gatsbyjs/gatsby/pull/9442#discussion_r228879230